### PR TITLE
Birmingham/West Midlands Class 3- Marcus Garvey - Git and GitHub - Week 2

### PR DESCRIPTION
**Pull request by garvey-marcus-code**
# Your Details

- Your Name: Marcus Garvey
- Your City: Birmingham
- Your Slack Name: Marcus Garvey

# Homework Details

- Module: Git and Github
- Week: 2

-----
[View rendered homework.md](https://github.com/garvey-marcus-code/GitHomeworkFixErrors/blob/main/homework.md)

### CODE REVIEW

**Comment by tenzyns** [comment]
I have finished my review. No error found.

File: homework.md [hunk 1]
```diff
@@ -44,70 +44,74 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-Microsoft Word
+Visual Studio Code.
 
 ## Question 17
 
 What do you use "Commit Messages" for?
 
-To record what changes you made to the code
+To give a detailed description of what you have done in the commit.
```
Comment (PeterAhern) [inline homework.md:111]: And also to explain the reason why you have made the changes.

File: homework.md [hunk 2]
```diff
@@ -44,70 +44,74 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
+It allows you to track changes to different versions of a file.
+
+It allows you to easily see the differences between multiple versions of a file.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You create a copy of the repository from somebody elses account on Github to your account on Github. This allows you to experiment with the repository and files without effecting the original repository.
```
Comment (PeterAhern) [inline homework.md:63]: Great answer

**Comment by MoTaKhi** [comment]
well done Marcus

**Pull request closed without merging**